client/src/components/WorkoutSubmitForm/WorkoutSubmitForm.module.css

```diff
@@ -44,14 +44,23 @@
   margin: 0 5px;
 }
 
+.new-workout-actions-privacy-container:hover {
+  cursor: pointer;
+  color: #6291e0;
+}
+
 .new-workout-actions-textarea {
-  margin: 5px 0;
-  border: none;
-  width: 100%;
-  height: 100px;
+  margin: 10px 0;
+  max-width: inherit;
   background: inherit;
   font: 1rem "Nunito Sans", sans-serif;
   color: #636363;
+  border: none;
+  resize: auto;
+  max-width: 100%;
+  min-width: 100%;
```

Comment:
this is to keep the size of the textarea within the bounds of its parent container